e2e/verifyocsp/thirdparty-certificates/download_chain.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Step 1: Get the leaf certificate
+openssl s_client -connect www.akamai.com:443 < /dev/null 2>&1 |  sed -n '/-----BEGIN/,/-----END/p' > leaf.pem
+
+# Step 2: Get the intermediate certificate
+openssl s_client -showcerts -connect www.akamai.com:443 < /dev/null 2>&1 |  sed -n '/-----BEGIN/,/-----END/p' > intermediate.pem
```

Comment:
I don't think the `www.akamai.com` cert _should_ pass validation, since it's not valid for code signing, only TLS:

```
$ openssl x509 -in e2e/verifyocsp/thirdparty-certificates/leaf.pem -text -noout
...
            X509v3 Extended Key Usage: 
                TLS Web Server Authentication, TLS Web Client Authentication
```

I'm not sure off the top of my head where we could easily obtain a valid code signing chain. By definition, reaching out to a TLS server will get us a certificate for that rather than code signing.

To move forward, I propose we remove the third party certificate stuff for now, and the e2e test case that uses it, and address that separately. Does that sound like a plan?

Reply:
This part follows the discussion on
https://github.com/sylabs/singularity/issues/1152#issuecomment-1332183114

For passing the validation I had to append the `x509.ExtKeyUsageClientAuth`
https://github.com/sylabs/singularity/blob/3081ce65c3dae52d6238e310bfe0dc73ef67725c/internal/app/singularity/verify.go#L163-L169

I could remove it, but then there is no way for testing that OCSP actually works. 

The existing certificate chain fails early (due to the ed25519 issue), which makes it impossible to test the OCSP codepath to the end. Alternatively, we could create an RSA-only certificate chain and let OCSP complete the validation. 

How would you like to proceed?